invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/Home/components/lists/IdleLoansList/__tests__/__snapshots__/IdleLoansList.js.snap

```diff
@@ -2011,7 +2011,7 @@ exports[`IdleLoansList tests should render loans 1`] = `
                 <SeeAllButton
                   disabled={false}
                   fluid={false}
-                  to="/backoffice/loans?q=(state:(PENDING) AND _updated:%7B*%20TO%202019-10-18%7D)"
```

Comment:
did you rebase lately ? because this should not happen with changing the date in the snapshot, there is a global date mock